Neos.Neos/Tests/Behavior/Features/Bootstrap/RoutingTrait.php

```diff
@@ -185,7 +186,9 @@ public function theMatchedNodeShouldBeInContentStreamAndOriginDimension(string $
         Assert::assertTrue($matchedNodeAddress->workspaceName->isLive());
         Assert::assertSame($nodeAggregateId, $matchedNodeAddress->aggregateId->value);
         // todo useless check?
```

Comment:
💡 this can and should be migrated at some point to use workspace names in the routing tests